packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -348,21 +361,18 @@ class RecoveryPhoneHandler {
   async exists(request: AuthRequest) {
     const { uid, emailVerified, mustVerify, tokenVerified } = request.auth
       .credentials as SessionTokenAuthCredential;
-    const payload = request.payload as unknown as { phoneNumberMask: number };
```

Comment:
It seemed like this endpoint could be hit with this `phoneNumberMask` option to bypass the verified session check so I removed it. Might be nice to share this `4` value referenced in settings now and a couple times in auth-server as the same const.